src/debugger/adapter/DebugAdapter.js

```diff
@@ -39,6 +38,17 @@ class PrepackDebugSession extends DebugSession {
   _clientID: void | string;
   _adapterChannel: AdapterChannel;
 
+  _generateDebugFilePath(direction: "in" | "out") {
+    let time = Date.now();
+    let filePath = "/tmp/";
```

Comment:
This hard-code the fact that only works on Mac. I suggest you use tmp Node JS API:
https://www.npmjs.com/package/tmp

Reply:
I will add this in separate PR.